evap/settings.py

```diff
@@ -355,6 +367,16 @@
     "Enhances vibrant academic programs",
 ]
 
+
+### OpenID Login
+ACTIVATE_OPEN_ID_LOGIN = True
```

Comment:
false by default? because otherwise every developer might have to reprovision to get the new localsettings template?